docs/sources/installation/requirements.md

```diff
@@ -10,9 +10,11 @@ parent = "installation"
 weight = 100
 +++
 
-# Requirements
+# Grafana requirements
 
-This page includes useful information on the supported Operating Systems as well as the hardware requirements that are needed to install and use Grafana.
+This page lists the minimum hardware and software requirements to install Grafana.
+
+To run Grafana, you must have a supported operating system, hardware that meets or exceeds minimum requirements, a supported database, and a supported browser
 
 ## Supported operating systems
 
```

Comment:
What versions of the operating systems are supported? We should list that information on the requirements page.

Reply:
"All of them". So far we haven't received a lot of feedback of what's not supported so easier to keep it light than listing all possible versions supported?